docs/api/app.md

```diff
@@ -1278,28 +1278,22 @@ Returns `boolean` - Whether the current desktop environment is Unity launcher.
 ### `app.getLoginItemSettings([options])` _macOS_ _Windows_
 
 * `options` Object (optional)
-  * `path` string (optional) _Windows_ - The executable path to compare against.
-    Defaults to `process.execPath`.
-  * `args` string[] (optional) _Windows_ - The command-line arguments to compare
-    against. Defaults to an empty array.
+  * `type` string (optional) _macOS_ - Can be one of `mainAppService`, `agentService`, `daemonService`, or `loginItemService`. Defaults to `mainAppService`. Only available on macOS 13 and up. See [app.setLoginItemSettings](app.md#appsetloginitemsettingssettings-macos-windows) for more information about each type.
+  * `name` string (optional) _macOS_ - The name of the service. Required if `type` is non-default. Only available on macOS 13 and up.
```

Comment:
`name` exists on Windows as well and should be documented here.

Reply:
Addressed in [9ceb66e](https://github.com/electron/electron/pull/37244/commits/9ceb66eb74dfcbf80bc2e9445a27a7cbca095f57)